Assets/Scripts/Inventory/Character.cs

```diff
@@ -4,7 +4,14 @@ public class Character : MonoBehaviour, InventorySystemCharacter
 {
     private Inventory _inventory;
 
-    public Inventory Inventory => _inventory;
+    public Inventory Inventory
+    {
+        get
+        {
+            var inventory = _inventory;
+            return inventory;
+        }
+    }
```

Comment:
```suggestion
    public Inventory Inventory => _inventory;
```

its the same thing, dont make to too complicated

Reply:
done